lib/src/code_generator/type.dart

```diff
@@ -54,8 +54,29 @@ abstract class Type {
   /// Returns whether the dart type and C type string are same.
   bool get sameDartAndCType => sameFfiDartAndCType;
 
-  /// Returns the string representation of the Type, for debugging purposes
-  /// only. This string should not be printed as generated code.
+  /// Returns generated Dart code that converts the given value from its
+  /// DartType to its FfiDartType.
+  String convertDartTypeToFfiDartType(
```

Comment:
Please document the params so that if someone not familiar with objC introduces a new implementation of `Type` doesn't have to reverse engineer.

Reply:
Done.